ovh/data_hosting_privatedatabase_test.go

```diff
@@ -0,0 +1,54 @@
+package ovh
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccHostingPrivateDatabase_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckHostingPrivateDatabase(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccHostingPrivateDatabaseConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"data.ovh_hosting_privatedatabase.database",
+						"cpu",
+						"1",
+					),
+					resource.TestCheckResourceAttr(
+						"data.ovh_hosting_privatedatabase.database",
+						"datacenter",
+						"gra3",
+					),
```

Comment:
these values are to specific to your own database. you should maybe only test that the attribute is set, and not check its value

Reply:
fixed